tests/test_airmass_calculator.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from pyaraucaria.airmass_calculator import AirmassCalculator
+class TestAirmassCalculator(unittest.TestCase):
+
+    def test_calculate_airmass(self):
+        test_cases = [30.0, 45.0, 60.0, 75.0, 89.0]
+
+        for elevation in test_cases:
+            try:
+                result = AirmassCalculator.calculate_airmass(elevation)
+                expected_airmass = self.calculate_expected_airmass(elevation)
+                #asserting that the calculated airmass is close to the expected airmass
```

Comment:
I would suggest adding some additional assertions for the range of the result, especially, as we know that current implementation of the `calculate_expected_airmass` is not a one we can fully rely on.

For example, by definition, we know that airmass is the ratio of air masses relative to that at the zenith. Therefore, we know that it's 1.0 in the zenith direction and >1.0 in other directions. It also increases with the angle from the zenith (or conversely, decreases with the elevation angle). This  could be easy validated with assertions to ensure monotonic increase.

Reply:
Comment: Airmass less than 1 occurs when the sun is below the horizon, airmass equal to 1 happens when the sun is directly overhead, and airmass greater than 1 applies to all other daytime situations when the sun is above the horizon.

Changes:  included assertions to validate that the calculated airmass is within reasonable bounds and that it increases monotonically with the elevation angle. These assertions are more meaningful and appropriate for testing the airmass calculation, however it may not be reasonable to write assertions with fixed numeric bounds for airmass values since they can vary widely. What's the benchmark?